src/conda_deny_config.rs

```diff
@@ -79,16 +81,17 @@ impl CondaDenyTomlConfig {
         Ok(config)
     }
 
-    pub fn get_license_allowlists(&self) -> Vec<String> {
-        if self.tool.conda_deny.license_allowlist.is_none() {
-            Vec::<String>::new()
-        } else {
-            match &self.tool.conda_deny.license_allowlist {
-                None => vec![],
-                Some(LicenseAllowlist::Single(path)) => vec![path.clone()],
-                Some(LicenseAllowlist::Multiple(path)) => path.clone(),
-            }
-        }
+    pub fn get_license_allowlists(&self) -> Result<Vec<String>> {
+        let paths: Vec<String> = match &self.tool.conda_deny.license_allowlist {
+            None => return Ok(vec![]),
+            Some(LicenseAllowlist::Single(p)) => vec![p.clone()],
+            Some(LicenseAllowlist::Multiple(ps)) => ps.clone(),
+        };
+
+        let resolved: Vec<String> = expand_env_vars(&paths)
```

Comment:
```suggestion
        let resolved = expand_env_vars(&paths)
```
the type checker can do this for free